fastlane/lib/localizable_source.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+# Represents a localizable source for generating strings and downloading localizations from GlotPress.
+#
+# Example:
+#
+#   localizable = LocalizableSource.new(
+#     source_paths: ['/path/to/source1', '/path/to/source2'],
+#     localizations_root: '/path/to/localizations',
+#     gp_project_url: 'https://example.com/project'
+#   )
+#
+#   puts localizable.base_localization_strings
+#   # Output: /path/to/localizations/en.lproj/Localizable.strings
+#
+#   puts localizable.base_localization_strings(base_locale: 'fr')
+#   # Output: /path/to/localizations/fr.lproj/Localizable.strings
+#
+class LocalizableSource
+  # @return [Array<String>] An array of paths where source files related to localization are stored.
+  # This can include multiple directories where localized resources are kept.
```

Comment:
IMHO the terms "source files related to localization" and "localization resources" mades me think more about `.strings` files/resources than the `.swift` source files, so suggesting something like this instead:
```suggestion
  # @return [Array<String>] An array of paths to source files we want to scan for localization are stored.
  # This can include multiple directories where localized resources are kept.
```